tests/wpt/tests/css/css-transitions/pseudo-element-transition-continues-on-box-tree-re-construction.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>::before transition keeps updating while pointer moves to another element (IntersectionObserver)</title>
+<link rel="help" href="https://drafts.csswg.org/css-transitions/">
+<link rel="help" href="https://w3c.github.io/IntersectionObserver/">
+<meta name="assert" content="Moving pointer to another element does not freeze a ::before width transition on the host; host keeps expanding until it intersects the other element.">
```

Comment:
```suggestion
<link rel="help" href="https://github.com/servo/servo/issues/39225"/>
```
You could also add the issue reference.

Reply:
Thank you, I updated.